code/script.js

```diff
@@ -300,26 +412,39 @@ const filterCharacters = (keep) => {
       charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value))
   */
 
-  // Invoke a function to redraw the board with the remaining people.
-}
+// Invoke a function to redraw the board with the remaining people.
 
 // when clicking guess, the player first have to confirm that they want to make a guess.
 const guess = (personToConfirm) => {
-  // store the interaction from the player in a variable.
-  // remember the confirm() ?
-  // If the player wants to guess, invoke the checkMyGuess function.
-}
+  let playerGuess = confirm(`do you whant to guess on ${personToConfirm}?`);
+
+  if (playerGuess) {
+    checkMyGuess(personToConfirm);
+  } else {
+    alert("ERROR!you can contiue to guess!");
+  }
+};
 
 // If you confirm, this function is invoked
-const checkMyGuess = (personToCheck) => {
-  // 1. Check if the personToCheck is the same as the secret person's name
-  // 2. Set a Message to show in the win or lose section accordingly
-  // 3. Show the win or lose section
-  // 4. Hide the game board
-}
 
-// Invokes the start function when website is loaded
-start()
+const checkMyGuess = (personToConfirm) => {
+  if (personToConfirm === secret.name) {
+    winnerSound.play();
+    winoOrloseText.innerHTML = `YES!! You guess right ${personToConfirm} is correct!<img src=${secret.img} alt=${secret.name}>! You are a smart ass`;
+  } else {
+    EndSound.play();
+    winoOrloseText.innerHTML = `Error!! Te correct person was ${secret.name}.
```

Comment:
As an advice, try to be more careful with spelling next time on important displays like this, right now it is 'te' instead of 'the'